.eslintrc.js

```diff
@@ -3,10 +3,6 @@ require( '@rushstack/eslint-patch/modern-module-resolution' );
 module.exports = {
 	extends: [ './node_modules/newspack-scripts/config/eslintrc.js' ],
 	ignorePatterns: [ 'dist/', 'node_modules/' ],
-	globals: {
-		newspack_urls: 'readonly',
-		newspack_aux_data: 'readonly',
```

Comment:
According to the [failing job](https://app.circleci.com/pipelines/github/Automattic/newspack-multibranded-site/168/workflows/262de1b7-6cd3-4bc8-9607-f73aeb645134/jobs/675), if removing `newspack_aux_data` from this globals declaration, we need to include it as in a `/* global newspack_aux_data */` statement at the top of Brand.js.

Reply:
Oops, added in caff08e02f859d9f4169c3cda00d8f71a0d401fb